src/teleop/gui/src/components/LEDIndicator.vue

```diff
@@ -1,7 +1,9 @@
 <template>
   <div class="wrap">
     <span :class="['led', color]"></span>
-    <span class="name">{{ name }}</span>
+    <div v-if="show_name">
```

Comment:
I think adding this div here will lead to the name being put on the next line relative to the actual circle, maybe put the v-if directly in the span tag instead.